translations/ui-orders/en.json

```diff
@@ -512,6 +512,7 @@
   "poLine.cancellationRestrictionNote": "Cancellation description",
   "poLine.claimingActive": "Claiming active",
   "poLine.claimingInterval": "Claiming interval",
+  "poLine.donorOrganizationIds": "Donor information",
```

Comment:
here, translation value doen't match the key. Are you sure to set it this way?

Reply:
Can you elaborate a bit, please?

Reply:
there is no rule that translation key should match translation value, `donorOrganizationIds` is BE field that comes from history and we need a translation for it